### ISSUE TITLE
[Bug] Do not show active branch at the top on sidebar

### ISSUE THREAD

**Issue by chiahsien** [ForkIssues/Tracker#533]
Version: **Fork 1.0.74 on Mac Mojave, Git 2.20.1**

I choose to sort branches alphabetically, but the sorting seems to be broken.
By the way, I find it always moves the checked out branch to top, which is unexpected to me.

![2019-02-19 12 11 16](https://user-images.githubusercontent.com/1225437/52989830-22ecb400-3440-11e9-8f9f-3f04e80bc6bb.png)

**Comment by DanPristupov** [comment]
Fork just shows active branch first. I'll make an option to disable that. Previously `master` was always first and nobody worried :).

**Comment by YesThatAllen** [comment]
> Fork just shows active branch first. I'll add an option to disable that.

Thank you... it's very hard to follow as-is.

**Comment by DanPristupov** [comment]
OK, it looks like the new branch order is confusing for many users.

I've just published the new version with the usual sort order. However I will not force update for everyone. You can download the new version manually if you really can't stay with the current one :).

https://forkapp.ams3.cdn.digitaloceanspaces.com/mac/Fork-1.0.74.1.dmg

CC: @maksimovic @stk-ableton

**Comment by YesThatAllen** [comment]
Thank you for the quick help!

```
<Error>
<Code>AccessDenied</Code>
<BucketName>forkapp</BucketName>
<RequestId>tx00000000000001f8c228e-005c6c6dc8-e4a4bb-ams3a</RequestId>
<HostId>e4a4bb-ams3a-ams3</HostId>
</Error>
```

Perhaps it needs to be set world-readable?

**Comment by DanPristupov** [comment]
You are too fast :). CDN just got updated. Please try again (cache-free refresh might be required).

**Comment by YesThatAllen** [comment]
You are too fast!  Thank you for being so flexible & having a new build up so quickly.

Keep up the great work!

**Comment by DanPristupov** [comment]
I released the 1.0.74.2 hotfix for everyone.
